bin/README.md

```diff
@@ -1,22 +1,52 @@
-# portal-cli-app
-Simple CLI for displaying and publishing datasets
+## DataHub CLI
+
+This command line tool helps you to instantly build and deploy data-driven pages. It doesn't require you to know any frontend technologies (HTML, CSS, JS) so that you can focus on your data. Simply run a command (`datahub deploy`) and get a URL for your data page.
```

Comment:
Just pointing that datahub deploy command does not work yet. Do we want to add a note to indicate that.

Reply:
@risenW yes, I see and it should be OK (We'll develop it asap)